src/theme/global.scss

```diff
@@ -0,0 +1,112 @@
+@import "./variables.scss";
+@import "./components/exchange-coin-block.scss";
```

Comment:
this doesen't exists

Reply:
Yep, i tried split code to different blocks, mb forgot smthng

Reply:
Otherwise the project fails to start

Reply:
@8hokage searc.scss doesent exists neither